recipe/meta.yaml

```diff
@@ -26,8 +26,7 @@ requirements:
     - wheel
     # These are also needed at build time.
     - bleach 4.1.0
-    - bokeh 3.1.1 # [py<39]
-    - bokeh 3.2.0 # [py>=39]
+    - bokeh 3.3.0
     - pyct 0.5.0
     - param 1.13.0
```

Comment:
Is this correct?  It seems there is a mismatch between host and run requirements.